docs/using-baklava-in-react.stories.mdx

```diff
@@ -114,3 +114,55 @@ function MyComponent() {
 
 export default MyComponent;
 ```
+
+## Styling Components
+You can customize components with css variables or general theme variables.
+
+#### Inline CSS
+
+```jsx
+import { BlButton } from "@trendyol/baklava/dist/baklava-react";
+
+function MyComponent() {
+  const buttonStyle = {
+    "--bl-button-main-color": "rebeccapurple",
+    "--bl-button-main-hover-color": "purple",
+  }
+
+  return (
+    <BlButton style={buttonStyle}>button</BlButton>
+  );
+}
+
+export default MyComponent;
+```
+
+#### Styled Components
+
+```jsx
+import { BlTooltip, BlButton } from "@trendyol/baklava/dist/baklava-react";
+import styled from "styled-components"
+
+const Wrapper = styled.div`
+   .button-class {
+     --bl-button-main-color: red;
+   }
+
+   .tooltip-class {
+     --bl-tooltip-top: 100px !important;
```

Comment:
This is not a good example. You can not (should not) set tooltip position like that (at least for now).

Reply:
fixed, changed with color example